### ISSUE TITLE
Release v. Next

### ISSUE THREAD

**Issue by dblock** [ruby-grape/grape#2030]
Ongoing issue. If you feel like you need a new release because of changes in master, please add your +1. One of the maintainers will do it when they have time. A great way to accelerate this process is to send one of them a pony. Or coffee.

**Comment by stanhu** [comment]
@dblock Any chance you can get around to this soon?

**Comment by dblock** [comment]
This weekend unless someone beats me to it.

**Comment by dnesteryuk** [comment]
I will take care of that on this weekend, most likely, today :smile: 

![grape   RubyGems org   your community gem host](https://user-images.githubusercontent.com/128024/79039641-5b72c280-7beb-11ea-86b9-241506412fc2.png)

**Comment by stanhu** [comment]
Thank you @dnesteryuk and @dblock. 1.3.2 looks great. I think we can close this out.

**Comment by dnesteryuk** [comment]
I guess we can keep this thread, so people can request next release :wink:

**Comment by nbulaj** [comment]
There are some changes inroduced in 1.3.0 (and still valid for 1.3.2) that broke our production app after update.

We have app1 (Rails + Grape), app2 (Sinatra), frontend app. Frontend app communicates with app2 which in turn proxy requests to app1 using `curb` gem. When we used Grape 1.2.5 - everything was OK. But when we just changed version number in Gemfile and run the app with 1.3.0 - frontend app started to show infinite loader for requests. While debugging I found that Grape API start to respond to `curb` with Transfer-Encoding `chunked` O_o

I'll try to create a reproducable script for this, but don't sure how long it ill take. Maybe maintainers have an idea what could face such issue? I reviewed all the changes in git between 1.2.5 and 1.3.0 and didn't find the culprit :( Maybe I'll also try "by-commit" strategy to investigate

**Comment by dblock** [comment]
> And _app2_ starts to receive response headers with Transfer-Encoding `chunked` and never finish the request. Should I create an issue for that? I don't know if could create a reproducable example..

Thanks for bringing this up and for hunting down what changed. You should open a new issue either way. I think we need two things:

1. Find a workaround. You should figure out why your app stopped working and how to fix the app - IMO transfer-encoding and request not finishing are just symptoms, why would it break a client?
1. A new paragraph in UPGRADING once we figure out what the issue is.

**Comment by nbulaj** [comment]
OK I created a reproducable example with small readme on how to test it: https://gist.github.com/nbulaj/bd89ba366290033bf71c0f9b60927709

So before the changes I pointed above it worked, and with a new changes it doesn't

**Comment by dm1try** [comment]
> And app2 starts to receive response headers with Transfer-Encoding chunked and never finish the request

just a bit clarification, it starts to receive but unable to proxy those requests properly(the browser waits for a proper chunk).

without proxy/with proxy
<img width="454" alt="Screen Shot 2020-05-06 at 20 03 25" src="https://user-images.githubusercontent.com/486807/81206444-cccc4800-8fd4-11ea-863d-5e00cf5c9e9a.png">


> Transfer-Encoding is a hop-by-hop header, that is applied to a message between two nodes, not to a resource itself.
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Transfer-Encoding

**Comment by dm1try** [comment]
> You should figure out why your app stopped working and how to fix the app 

on a quick look, `grape` does not set `Content-Length` header on its side and relies on `rack`
so, according to the information above

before https://github.com/ruby-grape/grape/pull/1956 the payload was prepared with https://github.com/rack/rack/blob/master/lib/rack/content_length.rb

but after the response was changed to "chunked" one with https://github.com/rack/rack/blob/master/lib/rack/chunked.rb
 
this broke the proxy client, as it unable to properly forward chunked requests(either chunk by chunk or collect and send all chunks together;`Content-Length` and `Transfer-Encoding: chunked` cannot be used at the same time)

**Comment by nbulaj** [comment]
Yep, thanks for clarification @dm1try . Seems like you right about the reason. I created an issue for this (https://github.com/ruby-grape/grape/issues/2051) as this one is about releasing process and requests, so maybe let's move our discussion there. Thanks!

**Comment by magni-** [comment]
> I guess we can keep this thread, so people can request next release 😉

Would love to see 1.3.3 for https://github.com/ruby-grape/grape/pull/2040 🙇

**Comment by dblock** [comment]
1.3.3 was released 2020/05/23

**Comment by dim** [comment]
Time for a v1.3.4? The recent changes concerning Ruby 2.7 compatibility definitely deserve a release.

**Comment by dblock** [comment]
I'd like #2072 to go in at least.

**Comment by dramalho** [comment]
👍 as per demonstrated interest in https://github.com/ruby-grape/grape/issues/2075 :P

**Comment by robinboening** [comment]
Since #2072 got merged v1.3.4 should be unblocked, right? Any chance you could get a new release out to rubygems.org @dblock? ❤️

**Comment by dblock** [comment]
We want https://github.com/ruby-grape/grape/pull/2084 too.

**Comment by stanhu** [comment]
Yeah, could we get v1.3.4 to pull in #2084? That one is quite concerning.

**Comment by dblock** [comment]
I released 1.4.0.
